CMakeLists.txt

```diff
@@ -29,7 +26,7 @@ generate_messages(
 catkin_package(
   INCLUDE_DIRS include
   LIBRARIES virtual_cam_image_loader
-  CATKIN_DEPENDS message_runtime
+  CATKIN_DEPENDS message_runtime std_msgs
```

Comment:
Is there a reason why you include this? `std_msgs` is not included in any header. Also, see https://answers.ros.org/question/58498/what-is-the-purpose-of-catkin_depends/